Tests/Tests.Shared/NotificationTests.cs

```diff
@@ -32,8 +32,6 @@ namespace Tests.Database
     [TestFixture, Preserve(AllMembers = true)]
     public class NotificationTests
     {
-        private const int MillisecondsToWaitForCollectionNotification = 50;
```

Comment:
We should investigate if there's a real bug with autorefreshing realms - on .NET Framework apps, this seems to work.